### ISSUE TITLE
[Feature Request] A web UI for the update process

### ISSUE THREAD

**Issue by 0x6b-dev** [FOSSBilling/FOSSBilling#65]
https://github.com/boxbilling/boxbilling/issues/948

**Comment by Anuril** [comment]
I think this can be closed right?

**Comment by BelleNottelling** [comment]
Not really. Some things like clearing out old files out is done, but the system still doesn't have any live progress as to what's happening / what's been done during the update process

**Comment by Anuril** [comment]
oh. okay, I see, that makes sense
